portal-services/src/main/java/com/telecominfraproject/wlan/portal/controller/client/ClientPortalController.java

```diff
@@ -222,34 +222,35 @@ public ListOfClientSessions getAllClientSessionsInSet(@RequestParam int customer
     public PaginationResponse<ClientSession> getClientSessionsForCustomer(@RequestParam int customerId,
             @RequestParam(required = false) Set<Long> equipmentIds,    		
             @RequestParam(required = false) Set<Long> locationIds,    		
+            @RequestParam(required = false) String macSubstring,
```

Comment:
need to reflect this in the portal-services-openapi.yaml and in cmap-portal-services-openapi.yaml